### DISCUSSION TITLE
Does seq bind remove existing bindings prior to adding a new one?

### DISCUSSION THREAD

**Discussion by christian-townsend** [datalust/seq-tickets#1362] [Help] [answered]
I am wondering, does `seq bind-ssl --thumbprint="THUMBPRINT HERE"` remove existing bindings before adding the new one? Or do I need to do this myself with something like `netsh http delete sslcert ipport=443` ?

I would like to automate this process so am trying to accommodate for the scenario where we are renewing a certificate.

**Comment by nblumhardt** [comment, marked as answer]
Hi Christian,

Each call to `bind-ssl` will override any earlier bindings on the same port (or port/hostname combination). Let me know if you observe anything unexpected :-)

Best regards,
Nick

**Comment by christian-townsend** [comment]
@nblumhardt thank you very much.
